.github/workflows/documentation.yml

```diff
@@ -0,0 +1,65 @@
+# SPDX-FileCopyrightText: 2024 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+#
+# SPDX-License-Identifier: MIT
+
+name: Deploy doxygen to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Install Dependencies on Linux
+        run: |
+          sudo apt update -qq
+          sudo apt install -y doxygen
+
+      - name: Checkout sources
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Configure project
+        run: >
+          cmake -S . -B ./build -DKDBindings_DOCS=ON
```

Comment:
Can you add a `docs` preset to the `CMakePresets.json`, so this becomes `cmake --preset=docs`
and the build call becomes `cmake --build --preset=docs` or `cmake --build --preset=docs --target docs`

Reply:
ok